src/test/java/uk/gov/companieshouse/api/accounts/validation/StocksValidatorTest.java

```diff
@@ -181,7 +183,9 @@ void testSuccessfulCrossValidationForMultipleYearFiler() throws DataException {
 
     @Test
     @DisplayName("Cross validation fails with valid note for multiple year filer")
-    void testErrorReturnedCrossValidationForMultipleYearFiler() throws DataException {
+    void testErrorReturnedCrossValidationForMultipleYearFiler() throws DataException, ServiceException {
```

Comment:
These tests are now duplicates; they used to test `crossValidate` which has now been totally removed from the class. Shall we bin them?